.tmux.conf

```diff
@@ -981,6 +980,11 @@ run -b '[ -z "#{window_active}" ] && [ -z "#{version}" ] && tmux set display-tim
 #   tmux_conf_theme_prefix_bg=${tmux_conf_theme_prefix_bg:-none}
 #   tmux_conf_theme_prefix_attr=${tmux_conf_theme_prefix_attr:-none}
 #
+#   tmux_conf_theme_mouse=${tmux_conf_theme_mouse:-'🖱️'}                          # U+1F5B1
```

Comment:
I prefer `U+2191` as Emoi tend to trigger status line bugs when different components disagree on Unicode character wirds
```
#   tmux_conf_theme_mouse=${tmux_conf_theme_mouse-'↗'}                            # U+2197
```